scrapers/perfectgonzo.yml

```diff
@@ -0,0 +1,28 @@
+name: Perfect Gonzo
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - perfectgonzo.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //div[@class="col-sm-6 col-md-6 no-padding-left no-padding-right"]/h2
+      Date:
+        selector: //div[@class="col-sm-6 col-md-6 no-padding-left no-padding-right text-right"]/span
+        postProcess:
+          - replace:
+              - regex: 'Added|,'
+                with: ''
+          - parseDate: January 02 2006
```

Comment:
For single digit days there is no 0 in  the site so it should be `2` and not `02`
Also `,` can be parsed so its not necessary to remove it
Safer to use
```yaml
      Date:
        selector: //div[@class="row"]//span
        postProcess:
          - replace:
              - regex: "Added "
                with: ""
          - parseDate: January 2, 2006
```